thoth/package_extract/image.py

```diff
@@ -464,6 +465,24 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
+def _get_absolute_link(root_path: str, path: str, iter: int) -> str:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
+        return None
+    if os.path.islink(path):
+        readlink = os.readlink(path)
+        if len(readlink.split("/")) == 1:
+            next_link = os.path.join(os.path.dirname(path), readlink)
+        else:
+            if readlink[0] == "/":
```

Comment:
We are checking for `/` int the beginning because `os.path.join` would cannot join if the second parameter has '/' in the beginning.